packages/cli/test/commands/pull.test.ts

```diff
@@ -28,6 +28,40 @@ describe('pull', () => {
     expect(devFileHasDevEnv).toBeTruthy();
   });
 
+  it('should fail with message to pull without a link and without --env', async () => {
+    try {
+      process.stdout.isTTY = undefined;
```

Comment:
This is definitely not ideal. We'll want to refactor the commands to not directly reference global values (like `process.stdout`), but the approach in this PR seems fine for now.

Reply:
`client.input` mock SGTM 😄

This works for now though 👍